### PR TITLE
Medicine frequency screen cleanup

### PR DESCRIPTION
**Pull request by Honey14**

### CODE REVIEW

**Comment by Honey14** [comment]
Assigning it to Vinay since Sasikanth is on leave.

File: app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyUpdate.kt [hunk 1]
```diff
@@ -8,7 +8,7 @@ import org.simple.clinic.mobius.next
 class MedicineFrequencyUpdate : Update<MedicineFrequencyModel, MedicineFrequencyEvent, MedicineFrequencyEffect> {
   override fun update(model: MedicineFrequencyModel, event: MedicineFrequencyEvent): Next<MedicineFrequencyModel, MedicineFrequencyEffect> {
     return when (event) {
-      is SaveMedicineFrequencyClicked -> dispatch(SaveMedicineFrequency(event.medicineFrequency))
+      is SaveMedicineFrequencyClicked -> dispatch(SaveMedicineFrequency(model.medicineFrequency))
```
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyUpdate.kt:11]: With this, we can also remove the `medicineFrequency` property from the `SaveMedicineFrequencyClicked` class, right?

**Review by vinaysshenoy — CHANGES_REQUESTED** [review]
PR looks good, left one comment about an unnecessary property.

File: app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyUpdate.kt (hunk 1 above)
Comment (Honey14) [reply app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyUpdate.kt:11]: Fair enough, I'll update it

**Review by vinaysshenoy — APPROVED** [review]
LGTM!

**Pull request merged**